src/vs/workbench/contrib/extensions/electron-browser/extensionsViewlet.ts

```diff
@@ -430,6 +433,12 @@ export class ExtensionsViewlet extends ViewContainerViewlet implements IExtensio
 		return this.searchBox.getValue().replace(/@category/g, 'category').replace(/@tag:/g, 'tag:').replace(/@ext:/g, 'ext:');
 	}
 
+	protected saveState(): void {
+		const value = this.searchBox.getValue();
+		this.searchViewletState['query.value'] = value;
```

Comment:
@klymenkoo Please check if the search string is not a market place query. Otherwise, there will be a request to marketplace always when the workspace is open. Retain only installed specific search strings.

Reply:
Done ✅